### PR TITLE
Initial contribution of openHAB-cloud documentation

### PR DESCRIPTION
**Pull request by marziman**
Added initial docu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+
+
+
+## Getting started
+
+
+openHAB Cloud is mainly based on the following frameworks and technologies:
+
+
+* [Node.js](https://nodejs.org/en/) - Server-side Javascript-framework
```
Comment (marziman) [inline README.md]: I can add a note there under the technologies

File: README.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+
+If you got asked to continue, confirm with Yes (Y).
+
+
+Now you need install git and clone the openHAB Cloud repository to your
+preffered directory (here: ubuntu) with the following commands:
+
+```
+sudo apt-get install git
+cd /home/ubuntu/
```
Comment (marziman) [inline README.md]: i will remove the ubuntu user stuff

File: README.md [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+sudo git clone https://username@github.com/openhab/openhab-cloud.git
```
Comment (marziman) [inline README.md]: ok so let me remove sudo

File: README.md [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+```
+
+
+Enter your password to clone the openhabcloud repro and after the completed
```
Comment (marziman) [inline README.md]: ok

File: README.md [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+drwx------  2 ubuntu ubuntu 4096 Jun  4 16:30 .cache
+drwxr-xr-x 13 root   root   4096 Jun  4 17:06 openhabcloud
+-rw-r--r--  1 ubuntu ubuntu  675 Apr  9  2014 .profile
+drwx------  2 ubuntu ubuntu 4096 Jun  4 12:34 .ssh
+```
+
+
+
+To run openHAB Cloud you need to install the required software bundles/stacks.
+Within the main directory "openhabcloud" you need to run
```
Comment (marziman) [inline README.md]: done

File: README.md [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+Installing openHAB Cloud on Linux (ubuntu).
+
+
+First we need to make sure that the list of packages and dependencies
+from all repositories are up-to-date:
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
```
Comment (digitaldan) [inline README.md]: Remove dist-upgrade , not needed.

Comment (digitaldan) [inline README.md]: i would add
`apt-get install build-essential redis-server mongodb nginx`

Comment (digitaldan) [inline README.md]: `sudo apt-get install build-essential redis-server mongodb nginx`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+
+Now you should be ready with your instance and see the welcome information of ubuntu.
+
+
+
+As next step go to back to the [Quickstart](#quickStart) and follow the installation steps.
+
+
+
+Next we have to configure nginx as webserver and copy the openHAB Cloud nginx config, overriding the default config:
```
Comment (digitaldan) [inline README.md]: This part is not specific to EC2 and should be part of the general install notes.  I guess we can also mention that this step is optional as you could just run the node server by itself.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,381 @@
...
+```
+sudo service nginx restart
+```
+
+
+Now we are ready to start the openHAB Cloud service.
+
+We will run openHAB Cloud as background service:
+```
+sudo nohup node app.js &
```
Comment (digitaldan) [inline README.md]: this is not specific to EC2, i would remove it.  Also there is etc/openhabcloud.service which is a systemd startup script that can be used to start and stop the cloud service on Ubuntu.

**Pull request merged**